README.md

```diff
@@ -15,22 +15,35 @@ The critical code is in [api/controllers](/api/controllers).
 The `token` authorization model refers to the Auth0 JWT token scheme, and
 many calls look at group memberships defined by the Auth0 app.
 
-The calls in the `/user` paths are proxied to the Auth0 administration API.
-The calls in the `/case`, `/method`, `/organization` and the like are proxied
-to the ElasticSearch database.
-
 The code expects a file `.env` to be located in the root directory of the folder,
 defining the following environment variables:
-`AUTH0_CLIENT_ID` and `AUTH0_CLIENT_SECRET` as per the Auth0 application which
-is managing the users, and `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
-corresponding to the AWS account with permissions to access the ElasticSearch
-cluster.
+ * `AUTH0_CLIENT_ID` and `AUTH0_CLIENT_SECRET` as per the Auth0 application which
+is managing the users.  The AUTH0_CLIENT_SECRET doesn't have to be the real one for dev environments, it can be anything (e.g. 'thisisjunk')
+ * `BEARER_TOKEN` so you can run `npm test` with an authenticated user.  To generate
+   a bearer token, go to `jwt.io`, put in the `AUTH0_CLIENT_SECRET` you picked above (e.g. 'thisisjunk') in the 'verify signature' spot for it, and adjust the payload to look like:
```

Comment:
README is actually not up to date.  I now generate all of these in the test framework, so they aren't needed either.